kroxylicious-integration-test-support/src/test/java/io/kroxylicious/proxy/config/ConfigurationTest.java

```diff
@@ -126,14 +130,15 @@ public static Stream<Arguments> fluentApiConfigYamlFidelity() {
                                   demo:
                                     targetCluster:
                                       bootstrap_servers: kafka.example:1234
-                                      tls: {}
+                                      tls:
+                                        clientAuth: NONE
```

Comment:
These are upstream test cases.  The TLS peer is the client role so the clientAuth setting has no meaning.  It seems weird/confusing to see this configuration.

Reply:
Oh rubbish your right I'd missed they were upstream config.